app/models/contact_message.rb

```diff
@@ -0,0 +1,17 @@
+class ContactMessage
+  include ActiveModel::Model
+
+  attr_accessor :message, :name, :email
+  attr_reader :errors
+
+  def initialize(attributes = {}, options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.